src/main/java/org/mule/service/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -109,8 +111,16 @@
       getInteger(REQUEST_STREAMING_BUFFER_LEN_PROPERTY_NAME, DEFAULT_REQUEST_STREAMING_BUFFER_SIZE);
 
   // Stream responses properties
+  private static final String USE_WORKERS_FOR_STREAMING_PROPERTY_NAME =
+      SYSTEM_PROPERTY_PREFIX + "http.responseStreaming.useWorkers";
+  private static final boolean useWorkersForStreaming =
```

Comment:
use feature flagging component

Reply:
Sorry. Not implemented for service. Ignore.

Reply:
It's not possible for the moment because it's a change in the service (runtime level), and the feature flagging component is loaded with the apps